### ISSUE TITLE
tidal-hifi stops playback when starting mpv

### ISSUE THREAD

**Issue by C-O-D** [Mastermindzh/tidal-hifi#466]
As the title says...why is this?
Both running on pipewie and should be no problem. I can play TIDAL with KODI and mpv runs fine in parallel...

**Comment by Mastermindzh** [comment]
Hi,

TIDAL enforces a strict limit of playback on 1 simultaneous device whilst also being online.
See their official documentation for this topic:
https://support.tidal.com/hc/en-us/articles/201623252-How-Many-Devices-Can-I-Use-Simultaneously

So nothing we can do about it I'm afraid, sorry.

**Comment by C-O-D** [comment]
I wasn't talking about using TIDAL several times in parallel, but about looking at pictures with MPV while listening to music on tidal-hifi, for example...

**Comment by Mastermindzh** [comment]
Reopened the issue and corrected labels based on your last post.
I have no idea why it would pause, it's not something we actively check for/against.

**Comment by C-O-D** [comment]
Found out it has to do with the [mpv-mpris](https://github.com/hoyon/mpv-mpris) plugin - if I remove it (~/.config/mpv/scripts) playback doesn't break on tidal-hifi. vlc, for example, also supports mpris and continues to run without any problems - so what's the problem?

**Comment by Mastermindzh** [comment]
I don't know. I don't use mpv-mpris.

At some point I can probably start debugging it but I'm busy with other things now and this has a lower priority unfortunately.
